locale/en/knowledge/getting-started/control-flow/what-are-callbacks.md

```diff
@@ -51,4 +51,4 @@ The typical convention with asynchronous functions (which almost all of your fun
        //This code gets run after the async operation gets run
     });
 
-You will almost always want to follow the [error callback convention](/articles/errors/what-are-the-error-conventions), since most Node.js users will expect your project to follow them. The general idea is that the callback is the last parameter. The callback gets called after the function is done with all of its operations. Traditionally, the first parameter of the callback is the `error` value. If the function hits an error, then they typically call the callback with the first parameter being an Error object. If it cleanly exits, then they will call the callback with the first parameter being null and the rest being the return value(s).
+You will almost always want to follow the [error callback convention](../../errors/what-are-the-error-conventions), since most Node.js users will expect your project to follow them. The general idea is that the callback is the last parameter. The callback gets called after the function is done with all of its operations. Traditionally, the first parameter of the callback is the `error` value. If the function hits an error, then they typically call the callback with the first parameter being an Error object. If it cleanly exits, then they will call the callback with the first parameter being null and the rest being the return value(s).
```

Comment:
It might be better to have the absolute URL here instead of the relative one.

Reply:
Yes, please start with `/en/…`